### ISSUE TITLE
No need to retry minting if no suitable peers found everywhere

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldfoundation/tft#81]
It is retried in the handler of de deposit anyway on any error

**Comment by DylanVerstraete** [comment]
This issue blocks all other operations on the bridge and should be solved asap

**Comment by robvanmieghem** [comment]
I don't agree, The thing is the lower level handler should not return an error, just log it. The retry is not done everywhere,I would not have implemented the retry if it was. and if an error occurs, we should not continue.

**Comment by DylanVerstraete** [comment]
If any error occurs the bridge will just be stuck? I don't think that is a good solution

**Comment by robvanmieghem** [comment]
at least we know the error then, now it is logged and will be rolled out
